client/lib/react-helpers/index.js

```diff
@@ -15,5 +20,28 @@ export default {
 			React.createElement( ReduxProvider, { store: reduxStore }, reactElement ),
 			domContainerNode
 		);
+	},
+
+	removeSidebarMiddleware( options = {
+		section: null,
+		isFullScreen: null
+	} ) {
+		let sectionOptions = {
+			hasSidebar: false
+		};
+
+		if ( options.isFullScreen !== null ) {
+			sectionOptions.isFullScreen = options.isFullScreen;
+		}
+
+		// Return the middleware.
+		return ( context, next ) => {
+			// Remove the sidebar.
+			context.secondary = null;
+
+			context.store.dispatch( setSection( options.section, sectionOptions ) );
+
+			next();
+		}
```

Comment:
Thanks for noticing. I followed the structure of the code in that file with `renderWithReduxStore` being in `export default {}` so I put my fn there too. I don't know why we aren't using named exports here. Will refactor.